app/components/courses/no_results_component.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="app-search-results">
+  <% if devolved_nation? %>
```

Comment:
I've ended up in a bit of a funny state:
Searched for courses in Wales.. then changed it to a search across England and applied all the filters and the following was returned, still showing the Wales TT link:
![image](https://user-images.githubusercontent.com/47917431/132354623-e9bc7a49-24d9-4338-9176-6ec5ff704a0d.png)

Reply:
Good spot. I managed to get results for across England but look at the query params 🔍  

![image](https://user-images.githubusercontent.com/42515961/132356848-0ca4bae6-6e33-4ae7-821e-509fec708da7.png)

Reply:
yeah, appears to be retained all the way through from clicking change
![image](https://user-images.githubusercontent.com/47917431/132357558-dd3555cf-3cba-4619-b682-5803a76f447d.png)

Reply:
Yea i guess we either need to update/reset that when they click continue on the search page

Reply:
Good spot @davidgisbey @JR-G 
Think I will need to clear `lat` and `long` also

Reply:
That should be working now 👍 
https://github.com/DFE-Digital/find-teacher-training/pull/937/commits/e698d6cd9c845f02b497d3d89596665e9f6dd3c1